src/graphics/program-lib/standard.js

```diff
@@ -401,6 +413,7 @@ pc.programlib.standard = {
 
         code += varyings;
         code += chunks.basePS;
+        code += chunks.textureBordersPS;
```

Comment:
This doesn't look very good to me - you're injecting a UI only code to EVERY (even non-UI!) shader.